src/modules/blogs/blogs.controller.ts

```diff
@@ -39,4 +40,14 @@ export class BlogController {
       post: updatedBlog,
     };
   }
+
+  @skipAuth()
+  @ApiOperation({ summary: 'Get All Blogs' })
+  @ApiResponse({ status: 200, description: 'Blogs retrieved successfully', type: BlogResponseDto })
+  @ApiResponse({ status: 400, description: 'Page or Page_size must be a positive number' })
+  @Get()
+  async findAll(@Query('page') page: number = 1, @Query('page_size') page_size: number = 10) {
+    const result = await this.blogService.findAll(page, page_size);
+    return result;
+  }
```

Comment:
Use descriptive name

```suggestion
async getAllBlogs(@Query('page') page: number = 1, @Query('page_size') page_size: number = 10) {
    const result = await this.blogService.getAllBlogs(page, page_size);
    return result;
  }
```